components/slider/slider.tsx

```diff
@@ -137,6 +139,7 @@ const SliderComponent: React.FC<React.PropsWithChildren<SliderProps>> = ({
   const clickHandler = (event: React.MouseEvent<HTMLDivElement>) => {
     if (disabled) return
     if (!sliderRef || !sliderRef.current) return
+    if (isDrag) return setIsDrag(false)
```

Comment:
A new issue seems to be introduced here, when the user double-clicks and then drags, it may cause confusion in calculating the progress bar.
Steps to reproduce:
1. Double click to the slider dot;
2. Hold down the mouse on the second click and do not release it;
3. Drag the button;

Perhaps it would be better if we passed the `onClick` method directly to the element so that everything that triggers an event is passed directly to the user. We don't seem to need to do any hacking of the `onClick` event.

Reply:
Ok. :)